packages/cpsuLambda/src/schema/format_1/transformer.ts

```diff
@@ -79,9 +82,20 @@ export function populateTemplate(
   template: string,
   prescriptionItem: itemType,
   prescriptionDetails: requestType
-): Result<BundleEntry<Task>, string> {
+): Result<BundleEntry<Task>, string> | undefined {
   const entry = JSON.parse(template) as BundleEntry<Task>
 
+  if (prescriptionItem.status === "DispensingComplete") {
+    const forbiddenStatuses = ["NotDispensed", "Expired", "Cancelled"]
+
+    if (prescriptionItem.completedStatus && forbiddenStatuses.includes(prescriptionItem.completedStatus)) {
+      logger.info("Skipping data store update for DispensingComplete with NotDispensed or Expired status", {
```

Comment:
Does this need the other two forbidden statuses adding? Maybe reword it to clarify that it won't send that update on, to the PSU lambda, rather than mention the data store.

Reply:
Altered the logging message to be a bit more specific